scratchapi2/user.py

```diff
@@ -50,45 +56,63 @@ def info(self):
         self.loves = req["stats"]["loves"]
         self.favorites = req["stats"]["favorites"]
         self.comments = req["stats"]["comments"]
-        self.remix_number = req["stats"]["remixes"]
-        self.parent = Project(req["remix"]["parent"], getinfo=False)
-        self.root = Project(req["remix"]["root"], getinfo=False)
+        self.remixes = req["stats"]["remixes"]
+        self.parent = (
+            Project(req["remix"]["parent"], getinfo=True)

```

Comment:
`Project(req['remix']['parent'])` would raise a TypeError because if there is no parent then "parent" is `null`. I fixed this with a `Project... if req['remix']['parent'] else None` ternary expression.